packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1730,7 +1730,9 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
               actions: widget.actions ?? WidgetsApp.defaultActions,
               child: FocusTraversalGroup(
                 policy: ReadingOrderTraversalPolicy(),
-                child: child,
+                child: ShortcutRegistrar(
```

Comment:
This change does not look like a clean up. Is it intended?

Reply:
Ok I see you mentioning it in the PR. Maybe this should be the title of this PR.

Reply:
You're right, it's not really a "cleanup", but something that was missing from the previous PR that I committed with the `ShortcutRegistry`.  Without this, people will have to add their own registrar manually.

I've added a test to make sure it exists.